README.md

```diff
@@ -62,6 +62,7 @@ be on your way to contributing!
 
 ## Changelog
 
+* 2.47.7 - Fixed ConfidentialValidator to show a warning when the email example does not match
```

Comment:
Personal preference, the message dosent have to be the same but the start of it: `ConfidentialValidator - <desc>` is a nice format.
```suggestion
* 2.47.7 - `ConfidentialValidator` - Changed email violation to a warning 
```

Reply:
Sounds good!